### PR TITLE
refactor ut code

### PR DESCRIPTION
**Pull request by skeeey**
1. reorganize ut common functions
2. improve test coverage

### CODE REVIEW

File: pkg/helpers/testing/assertion.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+		t.Errorf("expected %q error, but got %q", expectedErr, actual.Error())
+		return
+	}
+	if len(expectedErr) == 0 && actual != nil {
+		t.Errorf("unexpected err: %v", actual)
+		return
+	}
+}
+
+func AssertActions(t *testing.T, actualActions []clienttesting.Action, expectedActions ...string) {
```
Comment (deads2k) [inline pkg/helpers/testing/assertion.go]: I recommend name change and doc. `expectedVerbs`?

**Comment by deads2k** [comment]
something broke in namespaces, but I like the idea

/approve
/assign @qiujian16

File: pkg/helpers/testing/assertion.go (hunk 1 above)
Comment (skeeey) [reply pkg/helpers/testing/assertion.go]: renamed and add the doc for each  assert method

**Pull request merged**